examples/camera/projection_zoom.rs

```diff
@@ -93,11 +80,13 @@ fn setup(
     ));
 
     commands.spawn((
-        Name::new("Cube"),
-        PbrBundle {
-            mesh: meshes.add(Cuboid::default()),
-            material: materials.add(Color::srgb(0.8, 0.7, 0.6)),
-            transform: Transform::from_xyz(1.5, 0.51, 1.5),
+        Name::new("Fox"),
+        SceneBundle {
+            scene: asset_server
+                .load(GltfAssetLabel::Scene(0).from_asset("models/animated/Fox.glb")),
```

Comment:
TIL of this syntax 👀 Is this better than `assert_server.load("models/animated/Fox.glb#Scene0")`?

Reply:
Not necessarily! Yanked straight from one of our other examples. Not sure if "better", possibly exposes to editor/LSP help? Since it's not just a string...

Reply:
I like it better because it's harder to typo.